src/main/java/com/mycompany/fusionauth/plugins/guice/MyExampleFusionAuthPluginModule.java

```diff
@@ -48,6 +49,7 @@ protected void configure() {
     // Below are Functional examples
     //
     passwordEncryptorMapBinder.addBinding("example-custom-md5").to(ExampleCustomMD5SaltedPasswordEncryptor.class);
+    passwordEncryptorMapBinder.addBinding("alt-custom-md5").to(ExampleCustomMD5SaltedAlternativePasswordEncryptor.class);
```

Comment:
All examples are prefixed with `example-`, so this could be `example-alternate-customer-md5`